### ISSUE TITLE
groups dataframe - Error in is.finite(x[[idx]]) : default method not implemented for type 'list'

### ISSUE THREAD

**Issue by j450h1** [Phippsy/brandwatchR#3]
Congrats on new package!

Small error here when I try to view the ```my_groups``` dataframe:

```
#install.packages('brandwatchR')
library(brandwatchR)
library(jsonlite)
library(dplyr)

brandwatch_credentials <- file.path("..", "credentials", "brandwatch.json")

bw <- jsonlite::fromJSON(brandwatch_credentials)

# Authenticate

bwr_auth(un= bw$USERNAME,
         pw= bw$PASSWORD,
         refresh = TRUE,
         cache = TRUE)

# Get projects
my_projects <- bwr_projects_get()
#head(my_projects)

PROJECT <- my_projects %>% filter(name == bw$PROJECT) %>% pull(id)

# Get groups
my_groups <- bwr_querygrp_get(project_id = PROJECT)
View(my_groups)
```

**Comment by Phippsy** [comment]
Thanks for the congrats and for flagging this. I'm on holiday until the end of next week but will take a look ASAP when I get back. Cheers.

**Comment by Phippsy** [comment]
To help troubleshoot - can you confirm if any query groups exist for your project `bw$PROJECT`?
If there are zero query groups, I suspect I need to add better error handling. If not, I'll have to see...

**Comment by j450h1** [comment]
No worries. I already have a working python script in place, but thought I might shift over to R with this package.  

Yes, there are 2 groups. The results are displayed when I just enter the variable name in console and execute; the problem is only when viewing the df, so it could be an Rstudio issue?

**Comment by Phippsy** [comment]
That's odd. I'm afraid I was unable to reproduce the error. 

I pulled querygroups for 2 projects - one with multiple querygroups, and one with zero querygroups. The project with querygroups shows as a normal DF when I call `View()`, and the project without displays an empty list when I call `View()`.


The only other instances of similar issues I could find with this error message were [here](https://github.com/haleyjeppson/ggmosaic/issues/9), relating to a version of `ggplot2`.

If you'd be willing to privately share an rds file or similar of your querygroup dataframe (`my_groups`), I'd happily take a look and see if I can reproduce. From the error messages I worked through, it looks like this may be caused by NA or NULL values in the object being displayed.
